src/articles_api_utils.js

```diff
@@ -0,0 +1,42 @@
+import axios from "axios";
+
+const ncNewsAPI = axios.create({
+  baseURL: "https://nc-news-api-ilr4.onrender.com/api/",
+});
+
+export const getArticles = (topic, sort_by, order, limit, p) => {
+  let query = "";
+  if (topic && topic !== "All") {
```

Comment:
We shouldn't worry about this logic for now. For this specific task, this bit isn't required. Try to only include logic relevant to that specific task at hand :)